Unscheduled Care/2. Unscheduled Care outputs.R

```diff
@@ -369,8 +427,59 @@ other_loc_emergency_adm <- emergency_adm %>%
   select(hscp_locality, data) %>%
   pivot_wider(names_from = hscp_locality, values_from = data)
 
+# Create objects for text emergency admissions by age group
+max_ea_age <- max(emergency_adm_age$financial_year)
+min_ea_age <- min(emergency_adm_age$financial_year)
+
+latest_ea_max_age <- emergency_adm_age %>%
+  filter(
+    year == max(year)) %>% 
+  filter(
+    data== max(data)
+  ) %>%
```

Comment:
```suggestion
  filter(
    year == max(year), 
    data == max(data)
  ) %>%
```

Reply:
Couldn't get this to work as I need it to filter year and then data. Think this tries both at same time?

Reply:
Yes, you're spot on, this is the equivalent of max(year) AND max(data). Unless those more technically astute than I know some tricks, the command might have to stay as original (rows 448 to 452).